scripts/windows/build_all.sh

```diff
@@ -18,8 +18,10 @@ sudo apt install -y gcc g++ gperf # Can be removed (deps are listed in docs/buil
 pushd ../../crypto_plugins/monero_c
     ./apply_patches.sh monero
     ./apply_patches.sh wownero
-    ./build_single.sh monero x86_64-w64-mingw32 -j$(nproc)
-    ./build_single.sh wownero x86_64-w64-mingw32 -j$(nproc)
+    sudo ./build_single.sh monero x86_64-w64-mingw32 -j$(nproc)
+    # WSL2 needs superuser permissions related to error:
+    # update-alternatives: error creating symbolic link ...: Permission denied
+    sudo ./build_single.sh wownero x86_64-w64-mingw32 -j$(nproc)
```

Comment:
SUDO=sudo ./build_single.sh wownero x86_64-w64-mingw32 -j$(nproc)

https://git.mrcyjanek.net/mrcyjanek/monero_c/src/commit/a032a20221579d5fb9445b370eec1b6bd4ff653b/build_single.sh#L87 (I need to change upstream for i686, but it is not used there)

Reply:
Sorry, I'm not sure how to integrate this comment?